src/test/java/science/icebreaker/integration/UserAccountCreation.java

```diff
@@ -0,0 +1,128 @@
+package science.icebreaker.integration;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.jsonwebtoken.JwtException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.Optional;
+
+import science.icebreaker.dao.entity.Account;
+import science.icebreaker.dao.entity.AccountConfirmation;
+import science.icebreaker.dao.repository.AccountConfirmationRepository;
+import science.icebreaker.dao.repository.AccountRepository;
+import science.icebreaker.data.request.RegistrationRequest;
+import science.icebreaker.util.mixins.JacksonMixins.IgnoreIdMixIn;
+import science.icebreaker.util.mock.RegistrationRequestMock;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestInstance(Lifecycle.PER_METHOD)
+public class UserAccountCreation {
```

Comment:
Is it on purpose to not name it `...Test`?

Reply:
Not really, I'll do that